taf/tools/repo/__init__.py

```diff
@@ -212,10 +213,15 @@ def update(url, clients_auth_path, clients_root_dir, from_fs, expected_repo_type
         the "test" target file), use --authenticate-test-repo flag. An error will be raised
         if this flag is omitted in the mentioned case. Do not use this flag when validating a non-test
         repository as that will also result in an error.
+
+        Some repositories can contain unauthenticated commits in-between two authenticated. The updater
+        will check if existance and order of all authenticated ones and ignore unauthenticated. To raise
+        an error if there new authenticated commits (newer than the last local commit), error-if-unauthenticated
```

Comment:
question: should this be "error if there **are** new **un**authenticated commits"?

Reply:
Yes